MAUI/Chat/suggestions.md

```diff
@@ -0,0 +1,396 @@
+---
+layout: post
+title: Suggestions in .NET MAUI Chat control | Syncfusion
+description: Learn here all about Suggestions support in Syncfusion .NET MAUI Chat (SfChat) control, its elements and more.
+platform: MAUI
+control: SfChat
+documentation: ug
+---
+
+# Suggestions in .NET MAUI Chat (SfChat)
+
+The `SfChat` control allows to display a list of options, as suggestions, either below the current message or at the bottom of the chat control. These suggestions can be presented for all the supported message types in `SfChat`. When a user selects a suggestion item, the `SuggestionItemSelected` event and `SuggestionItemSelectedCommand` will be triggered. Both the event and the command will provide the `SuggestionItemSelectedEventArgs` as arguments, providing the following information about the selected suggestion item.
+
+* `HideAfterSelection`: Gets or sets a value indicating whether the suggestions control should be shown or collapsed after an item is selected from the list of suggestions. The default value is true.
+* `Message`: Gets the instance of the current message for which suggestions was shown.
+* `SelectedItem`: Gets the item that was selected from the suggestion control.
+* `SuggestionType`: Gets a value that indicates whether the selected item is from an inline suggestion box or from an outline suggestion.
+
+Declaration
```

Comment:
Why "Declaration" word used here? what it mean?

Reply:
Removed the line